cdap-ui/app/cdap/components/PipelineDetails/PipelineRuntimeArgsDropdownBtn/RuntimeArgsKeyValuePairWrapper/RuntimeArgsRow.tsx

```diff
@@ -30,12 +30,31 @@ const styles = (theme): StyleRules => {
       display: 'grid',
       gridTemplateColumns: '50% 50%',
       gridGap: '10px',
+      '& legend': {
```

Comment:
Can you add a TODO to clean up these styles later once we verify that bootstrap doesn't override legend styles directly anymore.